js/Colors.js

```diff
@@ -0,0 +1,31 @@
+export default {
+    Transparent: 'transparent',
+    Wgite: '#ffffff',
+    Black: '#000000',
+    ExtraDarkGray: '#2d2d2d',
+    DarkGray: '#888e8e',
+    MediumGray: '#cacbcc',
+    VeryLightGray: '#dedede',
+    BaseBackground: '#f1f1f1',
+    Lightbox: 'rgba(0, 0, 0, 0.5)',
+    GorilaPrimary: '#10c0c6',
+    GorilaSecondary: '#4dde90',
+    FixedIncomePrimary: '#815ae8',
+    FixedIncomeSecondary: '#9278f4',
+    StocksPrimary: '#ed4581',
+    StocksSecondary: '#f766a1',
+    MultimarketPrimary: '#0dd6f4',
+    MultimarketSecondary: '#43e7ff',
+    CurrencyPrimary: '#fcbd1f',
+    CurrencySecondary: '#ffd364',
+    TangiblePrimary: '#d15130',
+    TangibleSecondary: '#e05f3e',
+    CashPrimary: '#314a61',
+    CashSecondary: '#456888',
+    Positive: '#75b629',
+    Negative: '#ef2e2e',
+    BuyPrimary: '#10c0c6',
+    BuySecondary: '#4dde90',

```

Comment:
Should these colors be duplicated? (they were already defined as GorilaPrimary and GorilaSecondary)

Reply:
yes